documentation/src/docs/metrics.md

```diff
@@ -308,58 +368,66 @@ public class MetricsConfig extends ComponentConfig {
 }
 ```
 
-# 4. Steps to add your own Metrics
-* Step 1. When start a segment store/controller service, start a Metrics service as a sub service. Reference above example in ServiceStarter.start()
+# Configuring Own Metrics
```

Comment:
@kevinhan88 section  heading changed...please check

Reply:
Is it a bit misleading to say "configuring metrics" instead of "creating metrics" or "coding metrics"?

Reply:
@kevinhan88 Accepted suggestion